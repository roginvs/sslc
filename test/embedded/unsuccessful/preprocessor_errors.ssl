#include <missing_file_1.ssl>
#include <missing_file_2.ssl>

#include "include_loop_1.ssl"

// This will not be reached
#include <missing_file_3.ssl>


procedure start begin
end