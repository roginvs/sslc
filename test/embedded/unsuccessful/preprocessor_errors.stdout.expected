Set include directory: ../include
Compiling preprocessor_errors.ssl
[Error] <preprocessor_errors.ssl>:1: Can't open include file "missing_file_1.ssl"
    #include <missing_file_1.ssl>
[Error] <preprocessor_errors.ssl>:2: Can't open include file "missing_file_2.ssl"
    #include <missing_file_2.ssl>
[Error] </unsuccessful/include_loop_1.ssl>:1: More than 256 nesting of #include
        from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from preprocessor_errors.ssl: 4:    #include "include_loop_1.ssl"
3 errors in preprocessor.
*** An error occured during preprocessing of preprocessor_errors.ssl ***
